stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -48,7 +48,7 @@
                 android:layout_height="wrap_content"
                 android:visibility="gone"
                 android:layout_marginVertical="2dp"
-                android:layout_marginHorizontal="10dp"
```

Comment:
Should we add these to style/StripePaymentSheetErrorMessage?
There's another usage of it in stripe_activity_payment_options.xml